packages/nc-gui/components/virtual-cell/components/ListChildItems.vue

```diff
@@ -148,7 +149,7 @@ const onClick = (row: Row) => {
           >
             <div class="flex items-center">
               <div class="flex-1 overflow-hidden min-w-0">
-                {{ row[relatedTableDisplayValueProp] }}
+                {{ renderValue(row[relatedTableDisplayValueProp]) }}
```

Comment:
check previous comment

Reply:
as spoken, this will be handled in another PR since it may test with other column types